### PR TITLE
fix(venmo): eligibility

### PR DESCRIPTION
**Pull request by mnicpt**
### Description
Incorrect eligibility condition for Venmo.

### Testing
1. venmoWebEnabled === false && shippingChange (Doesn't render Venmo button)
2. venmoWebEnabled === false && !shippingChange (Renders Venmo button)
3. venmoWebEnabled === true && shippingChange (Renders Venmo button)
4. venmoWebEnabled === true && !shippingChange (Renders Venmo button)

### CODE REVIEW

File: src/funding/venmo/config.jsx [hunk 1]
```diff
@@ -19,11 +19,23 @@ export function getVenmoConfig(): FundingSourceConfig {
 
     layouts: [BUTTON_LAYOUT.HORIZONTAL, BUTTON_LAYOUT.VERTICAL],
 
-    eligible: ({ experiment }) => {
+    eligible: ({ experiment, platform, shippingChange }) => {
+      if (experiment && experiment.enableVenmo === false) {
+        return false;
+      }
+
+      if (
```
Comment (mnicpt) [inline src/funding/venmo/config.jsx:27]: Don't render if shipping callback present and venmoWebEnabled === false.

File: src/funding/venmo/config.jsx [hunk 2]
```diff
@@ -19,11 +19,23 @@ export function getVenmoConfig(): FundingSourceConfig {
...
+        shippingChange
+      ) {
+        return false;
+      }
+
       if (
-        experiment &&
-        (experiment.enableVenmo === false ||
-          experiment.venmoWebEnabled === false)
+        platform === PLATFORM.DESKTOP &&
```
Comment (mnicpt) [inline src/funding/venmo/config.jsx:36]: Don't render if shipping callback and venmoWebEnabled === false.

**Review by jshawl — APPROVED** [review]
:shipit:

File: src/funding/venmo/config.jsx [hunk 3]
```diff
@@ -19,12 +19,12 @@ export function getVenmoConfig(): FundingSourceConfig {
...
-        experiment &&
-        (experiment.enableVenmo === false ||
-          experiment.venmoWebEnabled === false)
-      ) {
+    eligible: ({ experiment, shippingChange }) => {
+      if (experiment && experiment.enableVenmo === false) {
+        return false;
+      }
+
+      if (experiment.venmoWebEnabled === false && shippingChange) {
```
Comment (jshawl) [inline src/funding/venmo/config.jsx:27]: I think this needs `experiment &&` at the start of the condition to fix the typecheck issue

**Review by wsbrunson — APPROVED** [review]

**Review by ASISBusiness — COMMENTED** [review]
Working right

**Pull request merged**